### ISSUE TITLE
Game crashing on world creation

### ISSUE THREAD

**Issue by L31N0PTR1X** [tsunamayo/Starship-EVO#2860]
### Short Description of the issue:
Game crashes to windows after creating world.
...

### Steps to reproduce the issue:
Start a new game
...

### Environment
- Starship EVO version: 20w33a
- Graphics Card: Integrated 620

### Additional information:
[output_log.txt](https://github.com/tsunamayo/Starship-EVO/files/5156203/output_log.txt)
_(Screenshots, Videos, Error logs, Related issues)_

Please also include the `output_log.txt` file. You can find your log here: `%UserProfile%\AppData\LocalLow\Moonfire Entertainment\Starship EVO\`

**Comment by tsunamayo** [comment]
Yes it seems that this GPU has some issue for now, it is below minimum spec. I will try to make it work if I can get my hand on a PC with such a card. Thanks for your understanding

**Comment by tsunamayo** [comment]
By the way what is your laptop model? Thanks

**Comment by L31N0PTR1X** [comment]
Thanks! Keep me updated. If it helps, stormworks runs fine on my laptop and the art style is similar.Sent from my Samsung Galaxy smartphone.
-------- Original message --------From: Francois Duret <notifications@github.com> Date: 02/09/2020  02:49  (GMT+00:00) To: tsunamayo/Starship-EVO <Starship-EVO@noreply.github.com> Cc: L31N0PTR1X <l31n0ptr1x@gmail.com>, Author <author@noreply.github.com> Subject: Re: [tsunamayo/Starship-EVO] Game crashing on world creation (#2860) 
Yes it seems that this GPU has some issue for now, it is below minimum spec. I will try to make it work if I can get my hand on a PC with such a card. Thanks for your understanding

—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/tsunamayo/Starship-EVO/issues/2860#issuecomment-685233115",
"url": "https://github.com/tsunamayo/Starship-EVO/issues/2860#issuecomment-685233115",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by L31N0PTR1X** [comment]
It's an HP Pavillion G6 I think. Specs are as follows:Intel core i3 7th gen8gb ramIntel Integrated graphics 620Sent from my Samsung Galaxy smartphone.
-------- Original message --------From: Francois Duret <notifications@github.com> Date: 02/09/2020  02:50  (GMT+00:00) To: tsunamayo/Starship-EVO <Starship-EVO@noreply.github.com> Cc: L31N0PTR1X <l31n0ptr1x@gmail.com>, Author <author@noreply.github.com> Subject: Re: [tsunamayo/Starship-EVO] Game crashing on world creation (#2860) 
By the way what is your laptop model? Thanks

—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/tsunamayo/Starship-EVO/issues/2860#issuecomment-685233725",
"url": "https://github.com/tsunamayo/Starship-EVO/issues/2860#issuecomment-685233725",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by Pideorni** [comment]
I have what sounds like a really similar problem. GPU usage spikes in loading, i freeze on first frame in the world and then I crash.


Starship EVO version 20w33a

GPU: Geforce GTX 1080

[output_log.txt…]()

**Comment by tsunamayo** [comment]
@L31N0PTR1X I am not sure on how stormworks works, but I am using DX11 and compute shader a lot. For example this is using GPU culling and GPU terrain generation.
I have a lot of issue with Intel 620, I am now looking if I can buy a laptop for cheap ;) ...
 Cheers

**Comment by RyzaGrey** [comment]
Hello,
I've just bought the game and have the same issue. The game crashes when making a new world, if I wait a moment I can interact with stuff but my screen is frozen, I can interact with the in-game menu, and what I think is the inventory (codex, etc).
Basically, I can interact with the menus but I can't turn my head, look around or move at all.
This is what I see:
![image](https://user-images.githubusercontent.com/70778885/92285761-6e24c580-ef05-11ea-83b3-aa0b62441955.png)
Specs:
![image](https://user-images.githubusercontent.com/70778885/92285423-8b0cc900-ef04-11ea-89a3-d505d1f6d617.png)
I am an inch below minimum, but I don't think that's a problem as my processor usage is always below 100%.
But I don't know why menu takes 70% processor, loading screen 20% and in-game only 40% usage.
FPS counter is above 60.
I also opened a tab that has this in it, might help you...
![image](https://user-images.githubusercontent.com/70778885/92285606-01a9c680-ef05-11ea-8d67-e59e50b9448a.png)

**Comment by tsunamayo** [comment]
@RyzaGrey Hi, so I would need the ouput.log to investigate.

Please also include the `output_log.txt` file. You can find your log here: `%UserProfile%\AppData\LocalLow\Moonfire Entertainment\Starship EVO\`

Thanks

**Comment by RyzaGrey** [comment]
[output_log.txt](https://github.com/tsunamayo/Starship-EVO/files/5178036/output_log.txt)
Here it is, sorry for not uploading it right away.

**Comment by tsunamayo** [comment]
@RyzaGrey Hi, this issue is really weird your GPU is definitely good enough to run the game.
Could you make sure your GFX drivers and windows are up to date? Thanks

**Comment by RyzaGrey** [comment]
I have just downloaded and installed the latest NVIDIA driver, however I am not very experienced in this field, so I don't know if there are any other drivers that need to be kept up to date. Windows is up to date.
However, there is no good result.
This PC is a 2 year old gaming set, so there might be an issue with this, but I don't know if there are any other drivers that need to be managed besides this:
![image](https://user-images.githubusercontent.com/70778885/92412375-f1375d00-f14b-11ea-9e44-777befc3fc86.png)
Sorry for the peasant approach but I am really unexprerienced here... Thank you for your time.

**Comment by tsunamayo** [comment]
@RyzaGrey hi, this should be good enough. I will try to see if other people with GTX1660 have other issue and keep you posted. Thanks

**Comment by tsunamayo** [comment]
@RyzaGrey Hi, could you try to deactivate Head Bobbing in the Graphics Option menu? It seems that could be the culprit.
Tell me how it went.
Thanks

**Comment by RyzaGrey** [comment]
It works @tsunamayo 👍 .
![image](https://user-images.githubusercontent.com/70778885/92992432-34e2eb80-f4eb-11ea-8782-992ff4b89947.png)
´Thank you very much good luck developing the game!

**Comment by RyzaGrey** [comment]
One last thing: A week ago during the testing I created a world that went by the name "Ryza Grey". But the world creation went sour and the game crashed during the process. Now the game says that there is a world (or character) called "Ryza Grey", but I don't have it in my world selection unlike all the other worlds that I can simply delete and instantly make again a new world with the same character name.
It's just some mess that I've accidentally made, so if you see a world or a character called that, you may as well delete it right away :P.

**Comment by tsunamayo** [comment]
@RyzaGrey Hi, so you can simply go to the game save folder
C:\Users\Francois\AppData\LocalLow\Moonfire Entertainment\Starship EVO\Save_Data\Characters
and delete the old folder called Ryza Grey. That way you will be able to create a new world called like that.
Thanks
